cmd/util/azure.go

```diff
@@ -58,3 +58,28 @@ func ReadCredentials(path string) (*AzureCreds, error) {
 
 	return &result, nil
 }
+
+// ValidateMarketplaceFlags validates if any marketplace flag was used, all were set to a non-empty value
+func ValidateMarketplaceFlags(marketplaceFlags map[string]*string) error {
```

Comment:
fwiw there's also builtin MarkFlagsRequiredTogether cc @stevekuznetsov

Reply:
Unless there's a desire to do that, I'd rather leave what I have. 

It looks like I would need to pass in cmd in several different functions just to use it here.